drv/ignition-api/build.rs

```diff
@@ -2,10 +2,22 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+use build_fpga_regmap::fpga_regs;
+use std::{fs, io::Write};
+
 fn main() -> Result<(), Box<dyn std::error::Error>> {
     idol::client::build_client_stub(
         "../../idl/ignition.idol",
         "client_stub.rs",
     )?;
+
+    let out_dir = build_util::out_dir();
+    let mut reg_map = fs::File::create(out_dir.join("ignition_controller.rs"))?;
+    write!(
+        &mut reg_map,
+        "{}",
+        fpga_regs(include_str!("ignition_controller.json"))?,
```

Comment:
Our other FPGA images also commit the bitstream file and a README indicating which Buildomat build it came from.  Even though Hubris isn't responsible for flashing the Ignition target FPGA, I think this would be a good idea – if only so folks know where to find the image!

Reply:
This is from the same build which generates the pieces in `sidecar-mainboard-controller`. I'll copy over the `README` and put a sentence in to describe it. This did originally live in that crate, but there is a circular dependency between them if we want to keep it there. This felt like a reasonable compromise.

Reply:
Okay, that makes sense.